src/webui/webapplication.cpp

```diff
@@ -774,6 +777,33 @@ void WebApplication::action_command_bottomPrio()
     BitTorrent::Session::instance()->bottomTorrentsPriority(hashes);
 }
 
+void WebApplication::action_command_setLocation()
+{
+    CHECK_URI(0);
+    CHECK_PARAMETERS("hashes" << "location");
+
+    QStringList hashes = request().posts["hashes"].split("|");
```

Comment:
> Sorry to have you feel this way, but then, how do I express my opinion? this is code review after all... ?

I can understand code review driven from a technical standpoint. Meaning, there is a more technically-sound way of doing something. Many of your reviews do address something technical. But pointing out changes that make the functionality inconsistent with the rest of the file/project do not make sense to me.

> I don't think this is the case here, splitRef() is only on Qt > 5.4 which is available just recently (Qt 5.5 minimum now).

As I said, I can understand wanting to make a change like this. But this PR does not seem like the appropriate place to do it. If the project wants to change from using `split()` to `splitRef()` then that should be done in another PR addressing all uses of it. I don't think it's the responsibility of someone very new to the project to move along the project's technical debt.